package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "webpack.config.js",
   "scripts": {
-    "start": "node server/server.js",
+    "start": "nodemon server/server.js",
```

Comment:
I thought we only want to use nodemon in a development type of manner, not for production use.

Reply:
Reverted back to node

Reply:
Looks good.